actionview/lib/action_view/helpers/url_helper.rb

```diff
@@ -548,6 +548,8 @@ def current_page?(options)
         request_uri = url_string.index("?") ? request.fullpath : request.path
         request_uri = URI.parser.unescape(request_uri).force_encoding(Encoding::BINARY)
 
+        url_string.sub!(%r{^/(.*)/$},'/\1')
```

Comment:
Would this be correct as `url_string.chomp!('/')`, or would it have to be `url_string.chomp!('/') if url_string.start_with?('/')` ?